RealmModelGenerator/EntitiesView.swift

```diff
@@ -8,24 +8,39 @@
 
 import Cocoa
 
+protocol EntitiesViewDataSource : NSObjectProtocol {
+    func numberOfRowsInEntitiesView(entitiesView:EntitiesView) -> Int
+    func entitiesView(tableView:NSTableView, viewForTableColmn tableColumn: NSTableColumn?, row: Int) -> NSView?
+}
+
+protocol EntitiesViewDelegate : NSObjectProtocol {
+    func addEntityInEntitiesView(entitiesView:EntitiesView)
+    func removeEntityInEntitiesView(entitiesView:EntitiesView, index:Int)
+    func entitiesView(entitiesView:EntitiesView, selectionDidChange index:Int)
```

Comment:
index should probably be of type Int? because we will want to keep track of when nothing became selected or the nil state